README.md

```diff
@@ -308,6 +308,24 @@ The telepresence background processes `daemon` and `connector` both produces log
 - macOS `~/Library/Logs/telepresence`
 - Linux `~/.cache/telepresence/logs`
 
+## How it works
+
+When Telepresence 2 connects to a Kubernetes cluster, it
+
+ 1. Ensures Traffic Manager is installed in the cluster.
+ 2. Looks for the relevant subnets in the kubernetes cluster.
+ 3. Creates a virtual network device called a _TUN device_ and names it `tel0`.
+ 4. Assigns a non-conflicting IP address to `tel0` for each of the subnets.
+ 5. Binds itself to `tel0` and starts listening for traffic from `tel0` and Traffic Manager.
+ 6. Starts listening for and serving DNS requests.
+
+When a locally running application makes a network request to a service in the cluster, Telepresence will resolve the name to an address within the cluster.
+The operating system then sees that the TUN device has an address in the same subnet as the address of the outgoing packets and sends them to `tel0`.
+Telepresence is on the other side of `tel0` and picks up the packets, injecting them into the cluster through a gRPC connection with Traffic Manager.
```

Comment:
The packets aren't really "injected". The data of the packages are tunneled over a stream (in packets that are much larger than the TCP packets). I suggest you use different wording here unless you want to go into detail on how that works. How about "The operating system then sees that the address is covered by one of the TUN device's subnets, which means that telepresence will route the traffic to the cluster through a gRPC connection with the Traffic Manager"?